smsdk/ma_session.py

```diff
@@ -44,7 +44,7 @@ def _get_records(
         self,
         endpoint: str,
         method: str = "get",
-        _limit: float = np.Inf,
+        _limit: int = np.Inf,
```

Comment:
Can not set an integer to Infinite.  Can only set it to `None` for an invalid number.